src/mug/ssl.gleam

```diff
@@ -0,0 +1,334 @@
+import gleam/bytes_builder.{type BytesBuilder}
```

Comment:
SSL and not TLS?

Reply:
It's named after the erlang module `ssl`. Their explanation for the name `ssl` is:

> The application's name is still SSL because the first versions of the TLS protocol were named SSL (Secure Socket Layer). However, no version of the old SSL protocol is supported by this application.
> 
> — *From the [erlang docs](https://www.erlang.org/doc/apps/ssl/ssl.html)*

I can change it to `tls` if you wish!

Reply:
We should use the names that makes more sense for the actual domain rather than naming it after the tech debt of an implementation detail 👍

Reply:
Alright, I'll change the module's name to `tls`!